test/e2e/syncer/fromhost/from_host.go

```diff
@@ -156,4 +170,57 @@ var _ = ginkgo.Describe("ConfigMaps are synced to host and can be used in Pods",
 		gomega.Expect(envs["ENV_FROM_DEFAULT_NS"]).To(gomega.Equal("one"))
 	})
 
+	ginkgo.It("Delete configmap in vCluster should be sync again by host", func() {
+		framework.ExpectNoError(f.VClusterClient.CoreV1().ConfigMaps(cmsVirtualNamespace).Delete(f.Context, cm2VirtualName, metav1.DeleteOptions{}))
+
+		gomega.Eventually(func() bool {
+			defaultCmValues, err := f.VClusterClient.CoreV1().ConfigMaps(cmsVirtualNamespace).Get(f.Context, cm2VirtualName, metav1.GetOptions{})
+			if err != nil {
+				return false
+			}
+			return defaultCmValues.Data["ENV_FROM_DEFAULT_NS"] == "one" && defaultCmValues.Data["ANOTHER_ENV_FROM_DEFAULT_NS"] == "two"
```

Comment:
I'd check here if `defaultCmValues.GetUID()` is different than the UID of the ConfigMap that got deleted (as we expect an object to get re-created in virtual)